src/clocksync/clocksync.hpp

```diff
@@ -53,15 +63,29 @@ class ClockSyncItf {
      *
      * @return aos::Error.
      */
-    virtual aos::Error Start() = 0;
+    virtual Error Start() = 0;
 
     /**
      * Synchronizes system clock.
      *
      * @param time current time to set.
      * @return aos::Error
      */
-    virtual aos::Error Sync(const aos::Time& time) = 0;
+    virtual Error Sync(const aos::Time& time) = 0;
+
+    /**
+     * Subscribes for clock sync notifications.
+     *
+     * @param subscriber clock sync notification subscriber.
+     */
+    virtual Error Subscribes(ClockSyncSubscriberItf& subscriber) = 0;
```

Comment:
```suggestion
    virtual Error Subscribe(ClockSyncSubscriberItf& subscriber) = 0;
```

Reply:
No, we don't use 's at the end of function verbs

Reply:
fixed